server/src/organizations/organizations.service.ts

```diff
@@ -1,19 +1,35 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, NotFoundException } from '@nestjs/common';
 import { CreateOrganizationDto } from './dto/create-organization.dto';
 import { UpdateOrganizationDto } from './dto/update-organization.dto';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Repository, DeleteResult } from 'typeorm';
 import { Organization } from './entities/organization.entity';
+import fetch from 'node-fetch';
 
 @Injectable()
 export class OrganizationsService {
   constructor(
-    @InjectRepository(Organization)
-    private organizationsRepository: Repository<Organization>,
+    @InjectRepository(Organization) private organizationsRepository: Repository<Organization>,
   ) {}
 
   async create(createOrganizationDto: CreateOrganizationDto): Promise<Organization> {
-    return await this.organizationsRepository.save(createOrganizationDto);
+    return this.organizationsRepository.save(createOrganizationDto);
+  }
+
+  async checkEIN(ein: number): Promise<object> {
```

Comment:
let's make a...
```
type PropublicaOrganization = {
  ein: number,
  name: string,
// etc. (only need to add the properties we're using for now, plz include link to the api docs' return object
}
```
to be used in the return type here, instead of `object`

Reply:
not sure if i mentioned this, but meant to...
not a requirement but this function isn't actually doing any checking anymore, just getting the propublicaOrg, so i'd vote for `getPropublicaOrg` naming rather than `checkEIN`

Reply:
copy that.